### ISSUE TITLE
Best practice for bar layouts across multiple characters?

### ISSUE THREAD

**Issue by wrightling** [arkayenro/arkinventory#1405]
Is there a best practice for having different bar mappings across multiple characters? Example: my disc priest has a bar for each gear set using rules like "outfit( "Disc PVE" )", but obviously want different rules for different bar assignments for other classes/characters.

I looked over the wiki and FAQ and am looking for a best practice here.  Related to #915

**Comment by arkayenro** [comment]
if youre using an outfit mod then those tend to only save the data per character, so using a rule like `outfit("Disc PVE")` is only ever going to work on a character that has an outfit with that name, and i doubt youre going to name any other classes outfit that so even if the data was shared it would never cause any impact.

the only issue you get is when you use an `outfit` rule and look at another toons data, that may get a bit wonky if they happen to have an item in that outfit but mostly the items wont display as being in an outfit (as it can only see outfits from the current toon, not the one youre display the backpack for)

**Comment by wrightling** [comment]
Fair point. The issue still stands in other ways though. For example having a bar with tailor-related items on my priest but wanting different profession based bars for other toons

**Comment by arkayenro** [comment]
profession based categories are based off your actual professions, and you can prioritise one over the other (which is character specific)

plus you put tailoring on a bar and it will work for any character

if youre wanting category to bar assignments to be character specific then change the blueprint for each character so they dont share a common layout
